src/wdio/automation-environment.ts

```diff
@@ -23,8 +23,8 @@ export class AutomationEnvironment {
     }
 
     public static getWorkingBrowser(): WebdriverIO.Browser {
-        if (this.workingBrowser) {
-            return this.workingBrowser;
+        if (this.mode === AutomationMode.Standalone) {
+            return <WebdriverIO.Browser>this.workingBrowser;
```

Comment:
Instead of doing this cast, can't the type be updated to include the null?
In fact, there are cases where what this method returns can be null.

Reply:
Applied suggestion